src/visiomode/database/__init__.py

```diff
@@ -0,0 +1,41 @@
+#  This file is part of visiomode.
+#  Copyright (c) 2024 Olivier Delree <odelree@ed.ac.uk>
+#  Distributed under the terms of the MIT Licence.
+
+import typing
+
+import visiomode.models as models
+from visiomode.database.CorruptedDatabaseError import CorruptedDatabaseError
```

Comment:
I'm not a huge fan of this style of importing, i.e. `from thing.Stuff import Stuff`. Ideally the modules should be structured so that you can either do `from thing import Stuff`, or for example errors go in their own module such that the imports become `from visiomode.database.errors import CorruptedDatabaseError, NoMathingDatabaseError` etc